src/deep_learning_container.py

```diff
@@ -24,6 +24,64 @@
 
 TIMEOUT_SECS = 5
 
+TOKEN = None
+
+def requests_helper(url, headers = None, timeout = 0.1):
```

Comment:
can you provide descriptions on the functions added - some of the names are pretty similar so it would be helpful to have high level descriptions